index.js

```diff
@@ -35,6 +35,18 @@ if (!config.sessionSecret) {
 	process.exit(0);
 }
 
+if (config.languagePackages.length > 0) {
```

Comment:
This if statement isn't needed, unless you want to do `if (config.languagePackages)` to check it exists.